recipes/libcurl/all/conanfile.py

```diff
@@ -337,6 +406,7 @@ def _build_with_cmake(self):
         cmake.build()
 
     def package(self):
+        os.chdir(self.source_folder)
```

Comment:
again, I have huge doubts about that. all our recipes work correctly without this, maybe there is a conan bug?

Reply:
maybe, I am not sure what the official conan point of view is.
you do not need it if you use `conan create` to run this, b
but if you use the 'developing package way' like it is in the manual, and/or specify the source folder as a command line argument, than conan is in the wrong working directory

Reply:
using the context manager, will raise a ticket as soon as I have time

Reply:
removed, no chdir anymoer